Source/JavaScriptCore/wasm/WasmTypeDefinition.cpp

```diff
@@ -536,30 +536,26 @@ bool RTT::isSubRTT(const RTT& parent) const
     return false;
 }
 
-TypeInformation::TypeInformation()
-{
-#define MAKE_THUNK_SIGNATURE(type, enc, str, val, ...) \
-    do { \
-        if (TypeKind::type != TypeKind::Void) { \
-            RefPtr<TypeDefinition> sig = TypeDefinition::tryCreateFunctionSignature(1, 0); \
-            sig->ref();                                                                    \
-            sig->as<FunctionSignature>()->getReturnType(0) = Types::type;                  \
-            if (Types::type.isV128())                                                      \
-                sig->as<FunctionSignature>()->setArgumentsOrResultsIncludeV128(true);      \
-            thunkTypes[linearizeType(TypeKind::type)] = sig.get();                         \
-            m_typeSet.add(TypeHash { sig.releaseNonNull() });                              \
-        }                                                                                  \
-    } while (false);
-
-    FOR_EACH_WASM_TYPE(MAKE_THUNK_SIGNATURE);
-
-    // Make Void again because we don't use the one that has void in it.
-    {
-        RefPtr<TypeDefinition> sig = TypeDefinition::tryCreateFunctionSignature(0, 0);
-        sig->ref();
-        thunkTypes[linearizeType(TypeKind::Void)] = sig.get();
-        m_typeSet.add(TypeHash { sig.releaseNonNull() });
-    }
+const TypeDefinition& TypeInformation::typeDefinitionForLLIntBuiltin(LLIntBuiltin builtin)
```

Comment:
nit: let's call this `signatureForLLIntBuiltin` instead.

Reply:
Changed.